cmd/ssh.go

```diff
@@ -192,16 +226,78 @@ func (cmd *SSHCmd) jumpContainer(ctx context.Context, devPodConfig *config.Confi
 	}
 
 	// tunnel to container
-	return tunnel.NewContainerTunnel(client, cmd.Proxy, log).Run(ctx, func(ctx context.Context, containerClient *ssh.Client) error {
-		// we have a connection to the container, make sure others can connect as well
-		unlockOnce.Do(client.Unlock)
+	return tunnel.NewContainerTunnel(client, cmd.Proxy, log).
+		Run(ctx, func(ctx context.Context, containerClient *ssh.Client) error {
+			// we have a connection to the container, make sure others can connect as well
+			unlockOnce.Do(client.Unlock)
+
+			// start ssh tunnel
+			return cmd.startTunnel(
+				ctx,
+				devPodConfig,
+				containerClient,
+				client.WorkspaceConfig().IDE.Name,
+				log,
+			)
+		})
+}
+
+func (cmd *SSHCmd) reverseForwardPorts(
```

Comment:
`reverseForwardPorts()` also looks really similar to `forwardPorts`

Reply:
As before yes they are similar but separate (opposite) concepts and I'd like to keep them separated